### PR TITLE
[TESTS] Implemented user repository, service, controller tests

### PR DESCRIPTION
**Pull request by Zel1oy**

### CODE REVIEW

**Review by SaiyaJ1N — CHANGES_REQUESTED** [review]
Fix magic numbers like Long or String cases

File: src/test/java/ua/mate/team3/carsharingapp/user/controller/UserControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    @Sql(
+            scripts = SqlFilePaths.PATH_FOR_DELETE_ALL_USERS_SQL_FILE,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
+    )
+    public void updateUserRoleById_CustomerToManager_ReturnsUserInfo() throws Exception {
+        UpdateUserRoleRequestDto userRole = new UpdateUserRoleRequestDto();
+        userRole.setRole(MANAGER_ROLE);
+        Role managerRole = new Role();
+        managerRole.setName(MANAGER_ROLE);
+        managerRole.setId(2L);
```
Comment (SaiyaJ1N) [inline src/test/java/ua/mate/team3/carsharingapp/user/controller/UserControllerTest.java]: Here

File: src/test/java/ua/mate/team3/carsharingapp/user/controller/UserControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    @Sql(
+            scripts = SqlFilePaths.PATH_FOR_DELETE_ALL_USERS_SQL_FILE,
+            executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD
+    )
+    public void updateUserRoleById_ManagerToCustomer_ReturnsUserInfo() throws Exception {
+        UpdateUserRoleRequestDto userRole = new UpdateUserRoleRequestDto();
+        userRole.setRole(CUSTOMER_ROLE);
+        Role customerRole = new Role();
+        customerRole.setName(CUSTOMER_ROLE);
+        customerRole.setId(1L);
```
Comment (SaiyaJ1N) [inline src/test/java/ua/mate/team3/carsharingapp/user/controller/UserControllerTest.java]: Here

File: src/test/java/ua/mate/team3/carsharingapp/user/repository/UserRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    private UserRepository userRepository;
+
+    @BeforeAll
+    static void beforeAll() {
+        expectedUser = new User();
+        expectedUser.setId(3L);
+        expectedUser.setFirstName("UserName 1");
+        expectedUser.setLastName("LastName 1");
+        expectedUser.setPassword("Password 1");
+        expectedUser.setEmail("email@gmail.com");
```
Comment (SaiyaJ1N) [inline src/test/java/ua/mate/team3/carsharingapp/user/repository/UserRepositoryTest.java]: Here

File: src/test/java/ua/mate/team3/carsharingapp/user/repository/UserRepositoryTest.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        Role role = new Role();
+        role.setName(Role.RoleName.ROLE_CUSTOMER);
+        role.setId(1L);
```
Comment (SaiyaJ1N) [inline src/test/java/ua/mate/team3/carsharingapp/user/repository/UserRepositoryTest.java]: Here

File: src/test/java/ua/mate/team3/carsharingapp/user/service/UserServiceTest.java [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+import ua.mate.team3.carsharingapp.model.Role;
+import ua.mate.team3.carsharingapp.model.User;
+import ua.mate.team3.carsharingapp.repository.RoleRepository;
+import ua.mate.team3.carsharingapp.repository.UserRepository;
+import ua.mate.team3.carsharingapp.security.AuthenticationService;
+import ua.mate.team3.carsharingapp.service.NotificationService;
+import ua.mate.team3.carsharingapp.service.impl.UserServiceImpl;
+
+@ExtendWith(MockitoExtension.class)
+public class UserServiceTest {
```
Comment (SaiyaJ1N) [inline src/test/java/ua/mate/team3/carsharingapp/user/service/UserServiceTest.java:36]: And in this test class too

**Review by SaiyaJ1N — CHANGES_REQUESTED** [review]

**Review by SaiyaJ1N — APPROVED** [review]

**Review by padu12 — APPROVED** [review]

**Pull request merged**